models/multiphase/d2q9_pf_pressureEvolution/Dynamics.c.Rt

```diff
@@ -4,16 +4,6 @@
 // 
 // Maintainer: Travis-Mitchell @TravisMitchell
 //		Phase Field Implementation manipulated from: d2q9_pf - @mzdik
-
-// Code updates:
-//		19/07/16 - No dynamic grid refinement used, possibly discuss with Lukasz/Michal.
-//		05/08/16 - Typo error found in getMu - fixed
-//				 - MRT relaxation - new variables defined so that matrix multiplication not computed while updating R[0], R[1] etc.
-//      07/10/16 - Verification testing indicated an issue with the model (consistent factor of 2 over prediction)
-//				 - It was found that an error was made in the referenced paper in the given formula for the calculation
-//   			 - of macroscopic velocity. An additional c_s^2 was required on the forcing term.
-  
-   
```

Comment:
Sorry that one I should have kept, I had been deleting the comments that were above that one as they were unnecessary - but I will re-include the comment regarding the error in the paper.